slam_toolbox_ws/src/slam_simulation/launch/slam_sim.launch.py

```diff
@@ -66,10 +66,42 @@ def generate_launch_description():
         PythonLaunchDescriptionSource(gazebo_path)
     )
     
+    # TF Ref: 
+    # https://docs.ros.org/en/humble/Tutorials/Intermediate/Tf2/Writing-A-Tf2-Static-Broadcaster-Py.html#the-proper-way-to-publish-static-transforms
```

Comment:
Suggest adding more info such as:

- Add a comment on the dummy TFs:

  ```py
  # These static TF values are merely dummy values. The preferred way is to store the TFs in the URDF/SDF file and publish through `robot_state_publisher`.
  ```
- Add comment on the turtlebot SDF file:

  ```py
  # The links are defined based on the following SDF file:
  # https://github.com/ROBOTIS-GIT/turtlebot3_simulations/blob/humble-devel/turtlebot3_gazebo/models/turtlebot3_waffle/model.sdf
  ```

- Add comment on inspecting the SDF file:

  ```py
  # The links can be visualized in the TF section in rviz2 by the following commands:
  # $ export TURTLEBOT3_MODEL=waffle
  # $ ros2 launch turtlebot3_gazebo empty_world.launch.py
  # $ ros2 launch turtlebot3_bringup rviz2.launch.py
  # Ref: https://emanual.robotis.com/docs/en/platform/turtlebot3/simulation/
  # Ref: https://idminer.gitbook.io/robotis/part-1-turtlebot3/15.-ros2/15.2.-bringup-qi-yong/15.2.2.-rviz2-3d-ke-shi-hua-gong-ju
  ```

  If you know a better (or shorter) command to visualize TFs in rviz2, feel free to use yours instead.